lib/user_agent/browsers/apple_core_media.rb

```diff
@@ -29,27 +29,21 @@ def security
       end
 
       def os
-        if application
-          if application.comment[0] =~ /Windows NT/
-            OperatingSystems.normalize_os(application.comment[0])
-          elsif application.comment[2].nil?
-            OperatingSystems.normalize_os(application.comment[1])
-          elsif application.comment[1] =~ /Android/
-            OperatingSystems.normalize_os(application.comment[1])
-          else
-            OperatingSystems.normalize_os(application.comment[2])
-          end
+        return unless application
+
+        if application.comment[0] =~ /Windows NT/
+          OperatingSystems.normalize_os(application.comment[0])
+        elsif application.comment[2].nil?
+          OperatingSystems.normalize_os(application.comment[1])
+        elsif application.comment[1] =~ /Android/
+          OperatingSystems.normalize_os(application.comment[1])
         else
-          nil
+          OperatingSystems.normalize_os(application.comment[2])
         end
       end
 
       def localization
-        if application.nil?
-          nil
-        else
-          application.comment[3]
-        end
+        application.comment[3] if application
```

Comment:
I understand what you mean. It makes the ```platform```, ```os```, and ```localization``` methods have the same guard clause.

Still, don't you think a (simple) one-liner is better than an additional guard clause?